### ISSUE TITLE
Repeater beacon disable

### ISSUE THREAD

**Issue by IZ7AUH** [g4klx/YSFClients#105]
Hi,
I try to use YSFclient to make a link Room Wires-X with a YSFReflector but data is ok from Wires-x to YSF bur not from YSFReflector to Wires-x because on TX YSFClient send callsign users and information about a rpt in use, can possible disable it? how-to? best regards Frank IZ7AUH

**Comment by ChrisK9EQ** [comment]
I don't think this problem exists or it has been fixed. No problems bridging YSF to WiRES-X with recent code (late 2018, 2019).
